src/data/symbol/symbol.tsv

```diff
@@ -509,6 +512,82 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 記号	🀩	まーじゃん まーじゃんぱい ふゆ	麻雀牌	冬	SYMBOL
 記号	🀪	まーじゃん まーじゃんぱい ぱいたー じょーかー	麻雀牌	百搭	SYMBOL
 記号	🀫	まーじゃん まーじゃんぱい まーじゃんうら	麻雀牌	裏面	SYMBOL
+記号	🂠	とらんぷ とらんぷうら	トランプ	裏面	SYMBOL
+記号	🂡	とらんぷ すぺーど すぺーどA すぺーど1	トランプ	スペードA	SYMBOL
+記号	🂢	とらんぷ すぺーど すぺーど2	トランプ	スペード2	SYMBOL
+記号	🂣	とらんぷ すぺーど すぺーど3	トランプ	スペード3	SYMBOL
+記号	🂤	とらんぷ すぺーど すぺーど4	トランプ	スペード4	SYMBOL
+記号	🂥	とらんぷ すぺーど すぺーど5	トランプ	スペード5	SYMBOL
+記号	🂦	とらんぷ すぺーど すぺーど6	トランプ	スペード6	SYMBOL
+記号	🂧	とらんぷ すぺーど すぺーど7	トランプ	スペード7	SYMBOL
+記号	🂨	とらんぷ すぺーど すぺーど8	トランプ	スペード8	SYMBOL
+記号	🂩	とらんぷ すぺーど すぺーど9	トランプ	スペード9	SYMBOL
+記号	🂪	とらんぷ すぺーど すぺーど10	トランプ	スペード10	SYMBOL
+記号	🂫	とらんぷ すぺーど すぺーどJ すぺーど11	トランプ	スペードJ	SYMBOL
+記号	🂬	とらんぷ すぺーど すぺーどないと	トランプ	スペード ナイト	SYMBOL
+記号	🂭	とらんぷ すぺーど すぺーどQ すぺーど12	トランプ	スペードQ	SYMBOL
+記号	🂮	とらんぷ すぺーど すぺーどK すぺーど13	トランプ	スペードK	SYMBOL
+記号	🂱	とらんぷ はーと はーとA はーと1	トランプ	ハートA	SYMBOL
+記号	🂲	とらんぷ はーと はーと2	トランプ	ハート2	SYMBOL
+記号	🂳	とらんぷ はーと はーと3	トランプ	ハート3	SYMBOL
+記号	🂴	とらんぷ はーと はーと4	トランプ	ハート4	SYMBOL
+記号	🂵	とらんぷ はーと はーと5	トランプ	ハート5	SYMBOL
+記号	🂶	とらんぷ はーと はーと6	トランプ	ハート6	SYMBOL
+記号	🂷	とらんぷ はーと はーと7	トランプ	ハート7	SYMBOL
+記号	🂸	とらんぷ はーと はーと8	トランプ	ハート8	SYMBOL
+記号	🂹	とらんぷ はーと はーと9	トランプ	ハート9	SYMBOL
+記号	🂺	とらんぷ はーと はーと10	トランプ	ハート10	SYMBOL
+記号	🂻	とらんぷ はーと はーとJ はーと11	トランプ	ハートJ	SYMBOL
+記号	🂼	とらんぷ はーと はーとないと	トランプ	ハート ナイト	SYMBOL
```

Comment:
Is "たろっと" instead of "とらんぷ" better? As you know it's not included in common playing cards sold in Japan.
I think "はーとC", "かばろ", and "はーとかばろ" can be added too.